Product/Production/Adapters/PatientDiscovery_a0/src/main/java/gov/hhs/fha/nhinc/patientlocationquery/adapter/AdapterPatientLocationQuerySecured.java

```diff
@@ -46,7 +48,9 @@
     @Override
     public AdapterPatientLocationQueryResponseType adapterPatientLocationQuerySecured(
         AdapterPatientLocationQuerySecuredRequestType adapterPatientLocationQuerySecuredRequest) {
-        return new AdapterPatientLocationQueryResponseType();
+
+        AdapterPatientLocationQueryRequestType msg = new AdapterPatientLocationQueryRequestType();
```

Comment:
Why are we creating a new request, when we should be pulling it from the endpoint call?